### DISCUSSION TITLE
Twitter bot?

### DISCUSSION THREAD

**Discussion by Ainali** [govdirectory/website#177] [Ideas]
I saw the bot [EveryMPbot](https://twitter.com/everympbot) which tweets British MPs that are in Wikidata. Should we do something similar? It could either once a day tweet a really complete organization or ask for help with some missing information (or both).

**Comment by Abbe98** [comment]
That's a great idea, it could post "good data" regularly and every now and then post call to actions in the form of questions.

"Do you know about any social media accounts for XX?", "Do you know the official email address of XX?" ect

**Reply by Ainali** [reply]
Another idea, it would be pretty cool if the call to actions were in a local language.

**Comment by Abbe98** [comment]
I wonder if we can get away with a SPARQL query populating something like the following:

```
Can you help us find the {account or contact point}, for {agency}?

#{country} #{field of work} #wikidata
```

**Comment by Ainali** [comment]
And on the other end we could select organizations that have all info in the "info box" populated and at least one social media account, for example https://www.govdirectory.org/sweden/Q953058/

```
In Govdirectory you can find contact information for public organizations around the world. 
For example, the {account type} for {agency} in {country}.

{govdirectory-link}
```

**Comment by Ainali** [comment]
For traceability, I created #250 for the "good data" type of tweets.
